src/com/palmergames/bukkit/towny/command/NationCommand.java

```diff
@@ -583,6 +585,7 @@ public Nation newNation(TownyUniverse universe, String name, Town town) throws A
 		nation.addTown(town);
 		nation.setCapital(town);
 		nation.setUuid(UUID.randomUUID());
+		town.setRegistered(System.currentTimeMillis());

```

Comment:
I think you want nation.setRegistered()....

Reply:
Omg im such a derp